src/stash/benches/sqlite.rs

```diff
@@ -7,6 +7,9 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+// TODO: Make this work with the async API.
```

Comment:
add someone's name? Someone else might need to sign off on commenting this out

Reply:
We have oodles of nameless TODOs. I'm not really interested in doing this one, so adding my name doesn't make much sense.